CHANGELOG.md

```diff
@@ -1,17 +1,18 @@
-## [v1.1.12] - 2025-10-15
 
 ### Changed
 
+## [v1.1.12] - 2025-10-15
+
+This version of HOSS explicitly adds SMAP string variables to the list of
+exclusions in the earthdata-varinfo configuration file.
+
+## [v1.1.11] - 2025-10-15
+
```

Comment:
The `### change`d is still in the wrong place I think. It was a subheading for 1.1.11 I think.